python/HeaderService/hutils.py

```diff
@@ -358,7 +358,7 @@ def update_records(self, newdict, extname):
                 self.update_record(keyword, value, extname)
                 LOGGER.debug("Updating {}".format(keyword))
             except Exception:
-                LOGGER.debug("WARNING: Could not update {}".format(keyword))
+                LOGGER.warning("WARNING: Could not update {}".format(keyword))
```

Comment:
I would prefer that the changes of warning levels are made in a separate commit distinct from the commit that adds the ability to override the log level.

Reply:
Reverted to debug.

Reply:
It looks like `logging` suggests `.exception()` be used for logging exception information.